src/main/scala/org/broadinstitute/dsde/firecloud/EntityClient.scala

```diff
@@ -355,11 +359,13 @@ class EntityClient (requestContext: RequestContext, modelSchema: ModelSchema)(im
                 workspaceNamespace: String, workspaceName: String, pfbRequest: PfbImportRequest): Future[PerRequestMessage] = {
 
     def callArrow = {
-      pipeline { Post(avroToRawlsURL, pfbRequest) }
+      val gzipPipeline = addHeader (`Accept-Encoding`(gzip)) ~> pipeline ~> decode(Gzip)
+      gzipPipeline { Post(avroToRawlsURL, pfbRequest) }
     }
 
     def callRawls(entity: HttpEntity) = {
-      pipeline {
+      val authPipeline = authHeaders(requestContext) ~> pipeline
+      authPipeline {
```

Comment:
Now Arrow gets gzip headers and rawls gets auth headers, and this is spelled out explicitly where the two services are called.